x-pack/plugins/security_solution/common/detection_engine/get_query_filter.ts

```diff
@@ -38,27 +38,19 @@ export const getQueryFilter = (
     ignoreFilterIfFieldNotInIndex: false,
     dateFormatTZ: 'Zulu',
   };
-
-  const enabledFilters = ((filters as unknown) as Filter[]).filter((f) => !isFilterDisabled(f));
-  /*
-   * Pinning exceptions to 'kuery' because lucene
-   * does not support nested queries, while our exceptions
-   * UI does, since we can pass both lucene and kql into
-   * buildEsQuery, this allows us to offer nested queries
-   * regardless
-   */
   // Assume that `indices.query.bool.max_clause_count` is at least 1024 (the default value),
   // allowing us to make 1024-item chunks of exception list items.
   // Discussion at https://issues.apache.org/jira/browse/LUCENE-4835 indicates that 1024 is a
   // very conservative value.
   const exceptionFilter = buildExceptionFilter({
     lists,
-    config,
     excludeExceptions,
     chunkSize: 1024,
-    indexPattern,
   });
-  if (exceptionFilter !== undefined) {
+
+  const enabledFilters = ((filters as unknown) as Filter[]).filter((f) => !isFilterDisabled(f));
```

Comment:
Is this part needed?

I see further down the stack it is implemented:
https://github.com/elastic/kibana/blob/master/src/plugins/data/common/es_query/es_query/from_filters.ts#L61

I thought at one point we had this, but then later someone removed it as it wasn't needed? A test would be conclusive, just checking in case I don't understand something.

Reply:
You're right. I added some more tests to make sure it ignores disabled filters, then removed this piece and it all still worked the same!